Cartography.xcodeproj/project.pbxproj

```diff
@@ -788,6 +788,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				APPLICATION_EXTENSION_API_ONLY = YES;
+				CODE_SIGN_IDENTITY = "Mac Developer";
```

Comment:
Our CI doesn't have development identities installed, and fails to build at codesign step if we leave it at is.
I'm building on Bitrise with Carthage.